src/org/zaproxy/zap/extension/autoupdate/ManageAddOnsDialog.java

```diff
@@ -291,7 +299,10 @@ private JPanel getUninstalledAddOnsPanel() {
 				getMarketPlaceScrollPane().setViewportView(getUninstalledAddOnsTable());
 			}
 
+			JPanel searchPanel = getSearchPanel(getUninstalledAddOnsTable());

```

Comment:
Same comment regarding the rename.
The filter panel should not be visible until the table is shown, it's odd seeing the filter panel alone with the "Check for Updates" button.